DMTN-107.tex

```diff
@@ -227,54 +229,60 @@ \subsection{Commissioning Data Templates}\label{ssec:potsol_comm}
+Preliminary considerations for optimizing science with commissioning-data templates is discussed further in Section \ref{sssec:potsol_comm_r}.
 
-\clearpage
-\subsubsection{Prioritize Single-Filter Template-Building During Commissioning}\label{sssec:potsol_comm_r}
 
-This section explores ways to use the 6-month commissioning phase to build (at least) single-filter templates over as wide an area as possible ($\gtrsim$50\% of the WFD main survey).
-The optimal filter debatable: it's likely to be $r$ or $i$-band, and the latter might have the advantage of not requiring airmass correction for DCR (Section 5.2.4 of \citeds{LDM-151}).
-Many surveys have done single-band transient searches in the past (e.g., PTF extragalactic focused on $r$-band), and it is the cheapest way to cover the widest area possible with templates, and thus release the most alerts on {\em unique events} during LOY1. 
 
-There are two basic options for building single-filter templates from commissioning data over $\gtrsim$50\% of the WFD main survey area: (1) create a new commissioning mini-survey and give it dedicated time, or (2) create a template-building program for the observation scheduler which can run as "filler" during commissioning.
+\clearpage
+\subsubsection{Optimizing Science with Commissioning-Data Templates}\label{sssec:potsol_comm_r}
+
+The plans for LSST commissioning surveys are described in \citeds{LSE-79}.
+When this Tech Note was written, \citeds{LSE-79} described Final Science Verification Mini-Survey 1 in two phases.
+First, a three week survey of $\sim10\%$ of the LSST WFD main survey area ($\sim$1600 $\rm deg^2$) with $\sim$15 visits per filter per field ($\sim$1 year equivalent depth), from which template images are generated.
+Then, after a six-week gap, the same three week survey is executed and data is processed with the Prompt pipeline to perform DIA and generate alerts (which might be distributed with high latency via non-streaming interfaces).
+If Mini-Survey 1 is executed prior to the start of operations, then multi-band templates for $\sim$1600 $\rm deg^2$ ($\sim$10\% of WFD) would already exist and could be used for alert generation in LOY1.
+\citeds{LSE-79} also described Mini-Survey 2 as $\sim$300 $\rm deg^2$ to a 10-year equivalent depth, and a $\sim$100 $\rm deg^2$ 20-year Depth Test as part of Early Science Verification.
+Additional sky area would be observed as a part of the other Key Performance Metrics during Early Science Verification.
+{\bf The main takeaway is that \citeds{LSE-79} described that {\it at least} $\sim14\%$ of the WFD main survey area could have six-filter templates at the start of LOY1.}
+
+The ability to generate six-filter templates and distribute alerts during LOY1 over $14\%$ of the WFD area would enables some science, but would not maximize it under this Tech Note's working definition of maximizing the number of alerts for unique objects in LOY1.
```

Comment:
of the WFD area would enable (remove 's') some

Reply:
fixed typo